tools/swift-demangle/swift-demangle.cpp

```diff
@@ -30,12 +30,16 @@
 
 #include <cstdlib>
 #include <string>
-#if !defined(_MSC_VER) && !defined(__MINGW32__)
+#if !defined(_WIN32) && !defined(__MINGW32__)
```

Comment:
I think that I would rather see this as

    #if defined (__unix__) || (defined(__APPLE__) && defined(__MACH__))